model/project_vars.go

```diff
@@ -64,28 +68,67 @@ type ProjectVars struct {
 	AdminOnlyVars map[string]bool `bson:"admin_only_vars" json:"admin_only_vars"`
 }
 
+// ParameterMappings is a wrapper around a slice of mappings between names and
+// their corresponding parameters kept in Parameter Store.
 type ParameterMappings []ParameterMapping
```

Comment:
This is more conceptual, but will the ParameterMapping's ParameterName be just a concat of the project ref id (or some analogous thing) + the ParameterMapping's Name? If it is, couldn't we just compute the ParameterName rather than store it as well?

Reply:
That makes a lot of sense! Ty!